docs/synapse/userguides/model_updates/update_v2_193_0.rst

```diff
@@ -0,0 +1,227 @@
+
+  ``url``
+    The primary URL associated with the bucket item.
+
+
+``inet:service:channel``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the channel.
+
+
+``inet:service:channel:member``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the channel membership.
+
+
+``inet:service:emote``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the emote.
+
+
+``inet:service:group``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the group.
+
+
+``inet:service:group:member``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the group membership.
+
+
+``inet:service:permission``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the permission.
+
+
+``inet:service:relationship``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the relationship.
+
+
+``inet:service:rule``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the rule.
+
+
+``inet:service:session``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the session.
+
+
+``inet:service:subscription``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the subscription.
+
+
+``inet:service:tenant``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the tenant.
+
+
+``inet:service:thread``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the thread.
+
+
+``it:dev:repo:issue:label``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the object.
+
+
+``it:host``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the object.
+
+
+``it:host:tenancy``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the object.
+
+
+``it:software:image``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the object.
+
+
+``risk:leak``
+  The form had the following property added to it:
+
+  ``recipient``
+    The identity which received the leaked information.
+
+
+
+******************
+Updated Interfaces
+******************
+
+``inet:service:object``
+  The property ``url`` has been added to the interface.
+
+
+
+*************
+Updated Types
+*************
+
+``inet:web:hashtag``
+  The regular expression used to validate hashtags has been relaxed.
+
+
+
+***********
+Light Edges
+***********
+
+``enabled``
+    When used with a ``risk:leak`` and an ``risk:leak`` node, the edge
```

Comment:
```suggestion
    When used with a ``risk:leak`` and a ``risk:leak`` node, the edge
```

I know this was automatically generated but it sounds a little weird when both nodes are the same form.